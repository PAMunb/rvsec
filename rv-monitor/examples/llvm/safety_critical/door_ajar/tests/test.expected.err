Attempting to start the engine with door open!
