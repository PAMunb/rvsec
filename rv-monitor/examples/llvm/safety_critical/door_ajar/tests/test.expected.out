Door opened
Door closed
Attempting to start engine
Door opened
Attempting to start engine
Door closed
